### ISSUE TITLE
Wrongly getting a ``D417`` from a properly documented function

### ISSUE THREAD

**Issue by s0undt3ch** [PyCQA/flake8-docstrings#21]
Consider the following snippet:
```python
def terminate_process_list(
    process_list: List[psutil.Process], kill: bool = False, slow_stop: bool = False
) -> None:
    """
    Terminate a list of processes.

    Arguments:
        process_list:
            An iterable of :py:class:`psutil.Process` instances to terminate

    Keyword Arguments:
        kill:
            Kill the process instead of terminating it.
        slow_stop:
            First try to terminate each process in the list, and if termination was not successful, kill it.
    """
```

I get the following error message from flake8-docstrings:
```
D417 Missing argument descriptions in the docstring
```

I have `docstring-convention = google` in my `setup.cfg`, so, why is this not being properly identified? Any clues?

**Comment by s0undt3ch** [comment]
The weird thing is, other functions, documented in the same way, don't throw an error

**Comment by s0undt3ch** [comment]
I have to document the ``None`` return type... :confused:

**Comment by asottile** [comment]
flake8 docstrings does not implement any checks
